tftrt/examples/image-classification/image_classification.py

```diff
@@ -535,7 +536,7 @@ def get_frozen_graph(
     # Load from pb file if frozen graph was already created and cached
     if cache:
         # Graph must match the model, TRT mode, precision, and batch size
-        prebuilt_graph_path = "graphs/frozen_graph_%s_%d_%s_%d.pb" % (model, int(use_trt), precision, batch_size)
+        prebuilt_graph_path = "graphs/frozen_graph_%s_%d_%s_%d.pbtrt" % (model, int(use_trt), precision, batch_size)
```

Comment:
Using `pbtrt` would mean it only caches TRT graphs but it can also be used to cache native TF graphs right?

Reply:
.pbtrt includes the trt graph, .pb does not